hybrid/pv_source.py

```diff
@@ -31,7 +31,11 @@ def __init__(self,
         system_model = Pvwatts.default("PVWattsSingleOwner")
 
         if 'fin_model' in pv_config.keys():
-            financial_model = pv_config['fin_model']
+            if isinstance(pv_config['fin_model'], Singleowner.Singleowner):
+                financial_model = Singleowner.from_existing(system_model, "PVWattsSingleOwner")     # make a linked model instead
+                financial_model.assign(pv_config['fin_model'].export())                             # transfer parameter values
+            else:
+                financial_model = pv_config['fin_model']
```

Comment:
As I look as these changes, I'm wondering if it would be better to have a function in `power_source` that does this generically? It could be like the below. I think this would help define concretely what the financial model needs to be able to do in order to be used in a PowerSource. While removing the duplicate implementation in each technology type.

```
def import_financial_model(model): 
    if isinstance(pv_config['fin_model'], Singleowner.Singleowner):
        financial_model = Singleowner.from_existing(system_model, "PVWattsSingleOwner")     # make a linked model instead
        financial_model.assign(pv_config['fin_model'].export())                             # transfer parameter values
    else:
        # do checks that the appropriate functions are available: "set_financial_inputs", "value", "assign", "simulate_financials" etc
        financial_model = pv_config['fin_model']
```

Reply:
I agree. Implemented.